### PR TITLE
updates to OF v0.9

### PR DESCRIPTION
**Pull request by frauzufall**

### CODE REVIEW

File: example/src/testApp.cpp [hunk 1]
```diff
@@ -45,7 +45,7 @@ void testApp::setup(){
 	ofEnableAlphaBlending();
 
 	ofPixels pixels;
-	ofBitmapStringGetTextureRef().readToPixels(pixels);
+	ofTexture().readToPixels(pixels);
 	ofSaveImage(pixels,"font.bmp");
```
Comment (arturoc) [inline example/src/testApp.cpp]: hey thanks! can you just remove this 3 lines, they don't make sense at all, it seems just something i was testing for something else and i quickly added it in this example but never removed it

Comment (frauzufall) [inline example/src/testApp.cpp]: done!

**Comment by arturoc** [comment]
thanks!

**Comment by frauzufall** [comment]
thank you for this addon!

**Pull request merged**